### DISCUSSION TITLE
Bina ve Müteahhit Veri Tabaı

### DISCUSSION THREAD

**Discussion by ramazansubasi** [acikkaynak/deprem-yardim-projesi#75] [Ideas]
Yıkılan binalar ve bu binaların mütahitlerinin veri tabanının oluşturulması. Şirket, kişi, şehir bazında sorumluların tespit ve kamuoyuna duyurulması için faydalı olabilir.

**Comment by ramazansancar** [comment]
Merhaba,
Fikriniz için teşekkürler. Ancak Legal ekibi benzer projeyi reddetti.
Bkz. https://github.com/acikkaynak/deprem-yardim-projesi/discussions/28 https://github.com/acikkaynak/deprem-yardim-projesi/discussions/37
